templates/security/security-standards.html

```diff
@@ -54,7 +54,7 @@ <h1>Security standards</h1>
       <div class="p-notification--information is-light">
         <div class="p-notification__content">
           <div class="p-notification__message">
-            FIPS 140-3 is now available for Ubuntu 22.04 LTS. <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">Learn more</a>
+            FIPS 140-3 is now available for Ubuntu 22.04 LTS. <a href="/blog/fips-140-3-for-ubuntu-22-04lts" target="_blank" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">Learn more</a>
```

Comment:
Not sure where the copy for this lives, couldn't find it in the doc, but I think we should add punctuation 
```suggestion
            FIPS 140-3 is now available for Ubuntu 22.04 LTS. <a href="/blog/fips-140-3-for-ubuntu-22-04lts" target="_blank" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">Learn more.</a>
```